distributed/scheduler.py

```diff
@@ -1133,7 +1137,7 @@ class TaskState:
     #: within a large graph that may be important, such as if they are on the critical
     #: path, or good to run in order to release many dependencies.  This is explained
     #: further in :doc:`Scheduling Policy <scheduling-policies>`.
-    priority: tuple[int, ...]
+    priority: tuple[int | float, ...] | None
```

Comment:
nit: (https://peps.python.org/pep-0484/#the-numeric-tower)
```suggestion
    priority: tuple[float, ...] | None
```